pkg/sse/events.go

```diff
@@ -0,0 +1,24 @@
+package sse
+
+import "strings"
+
+const (
+	// dataHeader is the sse string header for sse-data
+	dataHeader = "data: "
+
+	// pingEvent is the type of event ping
+	pingEvent = "ping"
+)
+
+// getMessage returns the parsed message from the sse-data
+func getMessage(s string) string {
+	if strings.HasPrefix(s, dataHeader) && !strings.Contains(s, pingEvent) {
```

Comment:
Could a data event contain the word `"ping"`? Maybe it's a subword or word from the installed application

Reply:
i've changed the parsing and checking of the message, this was not necessary after all, we nevermind ping events :D